### DISCUSSION TITLE
Error Updating Jellyfin

### DISCUSSION THREAD

**Discussion by Outsmart2383** [tteck/Proxmox#2730] [Help Needed] [answered]
```
root@jellyfin-lxc:~# apt update && apt upgrade
Hit:1 http://archive.ubuntu.com/ubuntu lunar InRelease
Hit:2 http://archive.ubuntu.com/ubuntu lunar-updates InRelease
Hit:3 http://archive.ubuntu.com/ubuntu lunar-security InRelease
Ign:4 https://repo.jellyfin.org/ubuntu lunar InRelease
Err:5 https://repo.jellyfin.org/ubuntu lunar Release
  404  Not Found [IP: 68.183.204.194 443]
Reading package lists... Done
E: The repository 'https://repo.jellyfin.org/ubuntu lunar Release' no longer has a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
root@jellyfin-lxc:~#
```

Tried running `apt update && apt upgrade` and I'm getting error `E: The repository 'https://repo.jellyfin.org/ubuntu lunar Release' no longer has a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.`

I'm not quite sure what the root cause of this is. I believe the command is trying to pull a new version from a repository, and it appears that repository is no longer there, but I don't know why? Have I just got unlucky, and the repository has moved, and the script needs updating. Or have I got something wrong on my end? Thanks!

**Comment by tteck** [comment, marked as answer]
If the default Linux distribution is not adhered to, script support will be discontinued.

**Reply by Outsmart2383** [reply]
Apologies, but it's a while since I deployed this, and the notes I took at the time don't seem to indicidate that I deviated from my normal practice of simply using your default scripts, but are you saying that in order for me to be seeing this error I must have done so?

**Reply by tteck** [reply]
yes, `ubuntu lunar`

**Reply by Outsmart2383** [reply]
Weird. Guess I'll have to do some digging then. Thanks.
